ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -538,44 +511,27 @@ export default class ConfirmTransactionBase extends Component {
       },
       () => {
         this._removeBeforeUnload();
-        metricsEvent({
-          eventOpts: {
-            category: 'Transactions',
-            action: 'Confirm Screen',
-            name: 'Transaction Completed',
-          },
-          customVariables: {
-            recipientKnown: null,
-            functionType:
```

Comment:
Earmarking this as potentially good to have in the next iteration of these events.